oauth2-server/src/main/java/com/clouway/oauth2/IssueNewTokenActivity.java

```diff
@@ -64,7 +64,12 @@ public Response execute(Client client, Identity identity, Set<String> scopes, Re
               .setSubject(identity.id())
               .setIssuedAt(instant.asDate())
               .setExpiration(new Date(accessToken.expirationTimestamp()))
-              .claim("identity", identityAsJson(identity).toString())
+              .claim("id", identity.id())
```

Comment:
id could be changed to sub as it's standard claim. I know that in the past id was used, but sub is better.